engine/apps/alerts/models/alert_receive_channel.py

```diff
@@ -305,6 +306,8 @@ class AlertReceiveChannel(IntegrationOptionsMixin, MaintainableObject):
     alert_group_labels_template: str | None = models.TextField(null=True, default=None)
     """Stores a Jinja2 template for "advanced label templating" for alert group labels."""
 
+    additional_settings: typing.Optional[ServiceNowSettings] = models.JSONField(null=True, default=None)
```

Comment:
Similar to previous comment, can't we try to define a more general settings type so eventually we can use this for any other integration? Settings should match the integration type, right? (ie. if this is a servicenow integration, here you could have `ServiceNowSettings` but not if the type was different)